tests/knocking_test.go

```diff
@@ -0,0 +1,194 @@
+// +build knock
```

Comment:
Prefer the MSC number please, as feature names are ambiguous (particularly if there are multiple competing MSCs for the same feature.

Reply:
Also please add the MSC number to the filename, e.g like https://github.com/matrix-org/complement/blob/kegan/msc2836/tests/msc2836_test.go

Reply:
Good point, much more scalable indeed.